src/site/content/en/blog/emscripten-embedding-js-snippets/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Embedding JavaScript snippets in C++ with Emscripten
+subhead: Learn how to embed JavaScript code in your WebAssembly library to communicate with the outside world.
+description: Learn how to embed JavaScript code in your WebAssembly library to communicate with the outside world.
+date: 2022-01-20
```

Comment:
```suggestion
date: 2022-01-17
```

@rachelandrew If this is looking good otherwise, then I'm happy to publish today.

Reply:
Or today :)

```suggestion
date: 2022-01-18
```